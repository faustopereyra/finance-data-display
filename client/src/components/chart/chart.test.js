import { shallow } from "enzyme";
import React from "react";
import Chart from "./chart.component";

it("expect to render Chart component", () => {
    const mockData = [{ price: 101, date: "Sun Sep 09 2001 3:46:40 AM" }, { price: 101, date: "Sun Sep 09 2001 6:33:20 AM" }]
    expect(shallow(<Chart data={mockData} />)).toMatchSnapshot();
})