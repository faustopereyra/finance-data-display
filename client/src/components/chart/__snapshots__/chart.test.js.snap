// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Chart component 1`] = `ShallowWrapper {}`;
